.github/workflows/ci.yml

```diff
@@ -15,18 +15,14 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
+          cache: pip
       # from pdm
       - name: Set Variables and install dep
         id: set_variables
         run: |
-          sudo apt-get install libxml2-dev libxslt-dev python3-dev 
+          sudo apt-get install libxml2-dev libxslt-dev python3-dev
```

Comment:
this can be dropped I think its my temp fix...I forgot that

Reply:
I'll keep this since they might be used by the pip install step later.